services/src/main/java/org/keycloak/authentication/forms/RegistrationRecaptcha.java

```diff
@@ -224,19 +231,29 @@ public String getHelpText() {
         property.setLabel("Recaptcha Site Key");
         property.setType(ProviderConfigProperty.STRING_TYPE);
         property.setHelpText("Google Recaptcha Site Key");
-        configProperties.add(property);
+        CONFIG_PROPERTIES.add(property);
         property = new ProviderConfigProperty();
         property.setName(SITE_SECRET);
         property.setLabel("Recaptcha Secret");
         property.setType(ProviderConfigProperty.STRING_TYPE);
         property.setHelpText("Google Recaptcha Secret");
-        configProperties.add(property);
-
+        CONFIG_PROPERTIES.add(property);
+
+        REFINED_LANGUAGE_CODES.add("zh-HK");
```

Comment:
It's not the responsibility of RegistrationRecaptcha to map the defined locale to a "refined locale". Please remove this and use userLocale.getLanguage() directly.

Reply:
Unfortunately if we'll use just ```getLanguage()``` then users will never be able to translate the ```reCAPTCHA``` to these 10 ```refined``` languages.
Please refer to the https://developers.google.com/recaptcha/docs/language